src/TLuaInterpreter.cpp

```diff
@@ -2121,11 +2008,7 @@ int TLuaInterpreter::startStopWatch(lua_State* L)
         // stopwatch) behaviour if only a single NUMERIC argument (ID) supplied:
         bool autoResetAndRestart = true;
         if (lua_gettop(L) > 1) {
-            if (!lua_isboolean(L, 2)) {
-                lua_pushfstring(L, "startStopWatch: bad argument #2 type (automatic reset and restart as boolean is optional with a numeric stopwatch id, got %s!)", luaL_typename(L, 2));
-                return lua_error(L);
-            }
-            autoResetAndRestart = lua_toboolean(L, 2);
+            autoResetAndRestart = getVerifiedBool(L, __func__, 2, "automatic reset and restart", true);
```

Comment:
This boolean is optional, and is ***only*** used when the first argument is numeric when a `false` prevents the previous (prior to Mudlet 4.4.0) behaviour of resetting and restarting - this (no reset and restart) always happens if a string is given as the first argument.
```suggestion
            autoResetAndRestart = getVerifiedBool(L, __func__, 2, "automatic reset and restart with a numeric stopwatch id", true);
```